packages/events/package.json

```diff
@@ -23,6 +23,7 @@
   },
   "dependencies": {
     "emittery": "^0.4.1",
-    "ora": "^3.4.0"
+    "ora": "^3.4.0",
+    "web3-utils": "^1.6.0"
```

Comment:
Should maybe make this match the version range we have elsewhere (since I noticed it shows up as new in `yarn.lock`)

Reply:
ah I see, I must have just used yarn to install it without checking the version

Reply:
That's what I'm here for